man/systemd-journald.service.xml

```diff
@@ -129,6 +135,42 @@ systemd-tmpfiles --create --prefix /var/log/journal</programlisting>
     <constant>EPIPE</constant> right from the beginning.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Namespaces</title>
+
+    <para>Multiple journal 'namespaces' may exist simultaneously, each defining its own, independent log
+    stream managed by its own instance of <command>systemd-journald</command>. Namespaces are independent of
+    each other, both in the data store and in the IPC interface. By default only a single 'default' namespace
+    exists, managed by <literal>systemd-journald.service</literal> (and its associated socket
+    units). Additional namespaces are created by starting an instance of the
+    <literal>systemd-journald@.service</literal> service template. The instance name is the namespace
+    identifier, which is a short string used for referencing the journal namespace. Service units may be
+    assigned to a specific journal namespace through the <varname>LogNamespace=</varname> unit file setting,
+    see <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry> for
+    details.</para>
+
+    <para>Journal namespaces are both a mechanism for logically isolating the log stream of projects
```

Comment:
Should this paragraph be first?

Reply:
This seems to first define what namespaces are before we talk about their co-existence with each other.

Reply:
will move